app/experimenter/experiments/views.py

```diff
@@ -151,6 +194,44 @@ def filter_search(self, queryset, name, value):
             .order_by("-rank")
         )
 
+    def experiment_date_field_filter(self, queryset, name, value):
+        # this custom method isn't doing anything. There has to
+        # be a custom method to be able to display the select
+        # filter that controls which date range we show
+        return queryset
```

Comment:
Ah that's annoying, oh well

Reply:
I guess I can't be 110% sure that this is true.. what do you think?

Reply:
I think... If you take the definition of this field out of the filterset altogether, and just declare it as a choice field on the form, then it'll still show up, show the choices, you'll still be able to look up the value from the filterset method, and then you won't need this method at all?  Give that a shot.

Reply:
I made a general PR comment that addressed my questions with this!